README.md

```diff
@@ -2,33 +2,76 @@
 
 1.
   * Create a Human class that takes in a name and age.
-  * Add the function `ageOneYear` that ages the human. 
+  * Add the function `ageOneYear` that ages the human.
   * Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it".
   * Create an instance of the Human class.
     * console log your humans age
     * call ageOneYear on your human
     * console log their age again.
     * call eating on your human.
-    
-2. 
+
+    class Human {
+     constructor(name, age){
+       this.name = name;
+       this.age = age;
+     }
+
+     ageOneYear(str){
```

Comment:
No argument is needed for this function! It doesn't hurt the function, just for future reference :)